airflow/www/security.py

```diff
@@ -505,6 +507,7 @@ def sync_roles(self):
 
         :return: None.
         """
+        # breakpoint()
```

Comment:
is it expected to keep this comment here?

Reply:
Removed